test/server/lib/HostedStorageManager.ts

```diff
@@ -33,6 +33,7 @@ import {createTmpDir, getGlobalPluginManager} from 'test/server/docTools';
 import {EnvironmentSnapshot, setTmpLogLevel, useFixtureDoc} from 'test/server/testUtils';
 import {waitForIt} from 'test/server/wait';
 import uuidv4 from "uuid/v4";
+import {IDocWorkerMap} from "app/server/lib/DocWorkerMap";
```

Comment:
As much as possible, try to order imports alphabetically, and use the local quote style (I realize it is already inconsistent in this file)